src/tls13.c

```diff
@@ -48,6 +48,8 @@
     #include <sys/time.h>
 #endif
 
+#include <wolfssl/wolfcrypt/settings.h>
```

Comment:
This can be removed. Its at line 44 now.

Reply:
I removed it.